PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
keep the default named modifiedElectrons.
Clone them as needed in a cff which defines the re-making sequence.